pom.xml

```diff
@@ -773,6 +773,15 @@
             </activation>
             <build>
                 <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <postgresql.latest.image>registry.redhat.io/rhel9/postgresql-16</postgresql.latest.image>
```

Comment:
Just curious if this shouldn't define latest version as it was recently changed. Saw that when looking at https://github.com/quarkus-qe/quarkus-test-suite/pull/2132

Reply:
My understanding was that motivation had been our util class expecting tag and this was easiest fix. I also expect @mocenas tested it so if it is failing he will come back :-)

Reply:
> My understanding was that motivation had been our util class expecting tag and this was easiest fix.

That is true. 

I completely forgot that adding that. AFAIK missing ":latest" does not break anything in this particular case, but I will add it anyway, just to be sure.